scripts/sysman/memory.yml

```diff
@@ -132,6 +132,21 @@ members:
       desc: "[out] The total allocatable memory in bytes (can be less than the `physicalSize` member of $s_mem_properties_t)"
 --- #--------------------------------------------------------------------------
 type: struct
+desc: "Extension properties for Memory State"
+version: "1.7"
+class: $sMemory
+name: $s_mem_ext_state_t
+details:
+    - "Memory Page Offline Metrics"
+members:
+    - type: uint32_t
+      name: memoryPageOffline
```

Comment:
Can we have this counter as part of a more granular RAS error category.
Something like ZES_RAS_ERROR_CAT_MEMORY_ECC_ERRORS ?

Reply:
Memory page offline is not RAS error. It is a SW functionality to handle ECC errors. That's the reason to keep this separate from RAS

Reply:
> Memory page offline is not RAS error. It is a SW functionality to handle ECC errors. That's the reason to keep this separate from RAS

agreed